### PR TITLE
fix(callbar-button-with-popover): show popover on demand and emit modal opened event vue3

### PR DESCRIPTION
**Pull request by ramya-dp**
# PR Title

show popover on demand and emit modal opened event

## :hammer_and_wrench: Type Of Change

- [X] Fix
- [ ] Feature
- [ ] Refactoring
- [ ] Documentation

## :book: Description

show popover on demand and emit modal opened event.

## :bulb: Context

show popover on demand and emit modal opened event. Required to auto open the Ai Assist panel. Ubervoice PR https://github.com/dialpad/firespotter/pull/30372

## :pencil: Checklist

- [X ] I have reviewed my changes
- [ ] I have added tests
- [X] I have added all relevant documentation
- [ ] I have validated components with a screen reader
- [ ] I have validated components keyboard navigation
- [ ] I have considered the performance impact of my change
- [ ] I have checked that my change did not significantly increase bundle size
- [ ] I am exporting any new components or constants in the index.js in the component directory
- [ ] I am exporting any new components or constants in the index.js in the root

## :camera: Screenshots / GIFs

No UI changed

### CODE REVIEW

**Comment by ramya-dp** [comment]
@iropolo this is the vue3 version of https://github.com/dialpad/dialtone-vue/pull/1016. Requires approval before merge. Please could you review

**Review by iropolo — CHANGES_REQUESTED** [review]
Seems to be all right but I can see this difference between the v2 version

image

Seems like the 'button' text is at the left, not at the bottom as the v2 version if it is not correct could you update it please?

File: recipes/buttons/callbar_button_with_popover/callbar_button_with_popover_default.story.vue
Comment (braddialpad) [inline recipes/buttons/callbar_button_with_popover/callbar_button_with_popover_default.story.vue]: $attrs.openPopover

**Review by braddialpad — CHANGES_REQUESTED** [review]

**Comment by ramya-dp** [comment]
> Seems to be all right but I can see this difference between the v2 version
> 
> image
> Seems like the 'button' text is at the left, not at the bottom as the v2 version if it is not correct could you update it please?

@iropolo good catch! Seems like this is an existing issue - https://vue.dialpad.design/vue3/?path=/story/recipes-buttons-callbar-button-with-popover--default. Since it wasn't introduced in this PR I've created a Jira to address this separately https://dialpad.atlassian.net/browse/DP-73940.

**Review by iropolo — APPROVED** [review]
Okay cool! looks good 🚀

**Comment by ramya-dp** [comment]
Merge is still blocked. I think @braddialpad needs to approve as well, please could you look at the updates?

**Review by braddialpad — APPROVED** [review]

**Pull request merged**
